### ISSUE TITLE
I can't build my iOS app with Xcode 12.4 

### ISSUE THREAD

**Issue by long1eu** [Instabug/Instabug-iOS#399]
```
Failed to build module 'Instabug' from its module interface; the compiler that produced it, 'Apple Swift version 5.3 (swiftlang-1200.0.29.2 clang-1200.0.30.1)', may have used features that aren't supported by this compiler, 'Apple Swift version 5.3.2 (swiftlang-1200.0.45 clang-1200.0.32.28)'
```

**Comment by yousefhamza** [comment]
Since you opened this new issue https://github.com/Instabug/Instabug-iOS/issues/400
Is this one resolved?

**Comment by long1eu** [comment]
They are very different issues

**Comment by long1eu** [comment]
I tried doing this.

![Screenshot 2021-01-29 at 10 55 25](https://user-images.githubusercontent.com/9283414/106254261-32c2c480-6221-11eb-9602-f1f111720cf2.png)

**Comment by yousefhamza** [comment]
@long1eu We have been shipping XCFrameworks for a while so you don't have to do that anymore.

I'm investigating the issue and will get back to you ASAP.

**Comment by yousefhamza** [comment]
@long1eu Can you try using "iOS" for all configurations in "Supported platforms" build settings and check if this error https://github.com/Instabug/Instabug-iOS/issues/400#issue-796618058 is still happening?

**Comment by yousefhamza** [comment]
Also make sure you update to the latest version of CocoaPods 🙏

**Comment by long1eu** [comment]
Will do. Thanks

**Comment by long1eu** [comment]
The result is the same.

```
ld: warning: ignoring file /Users/long1eu/IdeaProjects/fire42/mobile/ios/Pods/Instabug/Instabug.xcframework/ios-arm64_armv7/Instabug.framework/Instabug, missing required architecture x86_64 in file /Users/long1eu/IdeaProjects/fire42/mobile/ios/Pods/Instabug/Instabug.xcframework/ios-arm64_armv7/Instabug.framework/Instabug (2 slices)
    Undefined symbols for architecture x86_64:
      "_OBJC_CLASS_$_IBGReplies", referenced from:
          objc-class-ref in SwiftInstabugFlutterPlugin.o
      "_OBJC_CLASS_$_Instabug", referenced from:
          objc-class-ref in SwiftInstabugFlutterPlugin.o
         (maybe you meant: _OBJC_CLASS_$_InstabugFlutterPlugin)
    ld: symbol(s) not found for architecture x86_64
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
```

I'm using COCOAPODS: 1.10.1

**Comment by yousefhamza** [comment]
Is this error still happening after Cocoapods update?
> Failed to build module 'Instabug' from its module interface; the compiler that produced it, 'Apple Swift version 5.3 (swiftlang-1200.0.29.2 clang-1200.0.30.1)', may have used features that aren't supported by this compiler, 'Apple Swift version 5.3.2 (swiftlang-1200.0.45 clang-1200.0.32.28)'
-------

The second problem [here](https://github.com/Instabug/Instabug-iOS/issues/399#issuecomment-769723177) happens because Xcode is looking for "iOS Simulator" framework - x86 slice, in the "iOS device" framework:
> ld: warning: ignoring file /Users/long1eu/IdeaProjects/fire42/mobile/ios/Pods/Instabug/Instabug.xcframework/ios-arm64_armv7/Instabug.framework/Instabug, **missing required architecture x86_64** in file /Users/long1eu/IdeaProjects/fire42/mobile/ios/Pods/Instabug/Instabug.xcframework/**ios-arm64_armv7**/Instabug.framework/Instabug (2 slices)

It looks like an issue that's specific to your project & settings, can you reproduce this in another project? If you managed to do that please send it to us for investigation 🙏

**Comment by long1eu** [comment]
I've already told you I've updated cocoapods.

**Comment by yousefhamza** [comment]
@long1eu Noted, But there are 2 issues reported. I'm confirming whether both issues are still occurring after Cocoapods update or only the one you mentioned in the comment here: https://github.com/Instabug/Instabug-iOS/issues/399#issuecomment-769723177

Also, just to double-check, did you re-install pods after updating Cocoapods?
